corehq/couchapps/__init__.py

```diff
@@ -22,4 +22,5 @@
     'apps_with_submissions': settings.NEW_APPS_DB,
     'exports_forms_by_app': (None, settings.NEW_APPS_DB),
     'forms_by_app_info': (None, settings.NEW_APPS_DB),
+    'sync_logs_by_date': settings.SYNCLOGS_DB,
```

Comment:
since this view is mainly intended to be used for analyzing affected users during sync issues, we really only care about synclogs going forward.